app/src/main/java/com/klk/dailyjournal/SecondActivity.kt

```diff
@@ -0,0 +1,227 @@
+package com.klk.dailyjournal
+
+import android.Manifest
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.graphics.Color
+import android.net.Uri
+import android.os.Build
+import android.os.Bundle
+import android.os.Environment
+import android.provider.MediaStore
+import android.util.Log
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ActivityCompat
+import androidx.core.content.FileProvider
+import com.klk.dailyjournal.data.NoteEntity
+import com.klk.dailyjournal.data.NoteRepository
+import com.klk.dailyjournal.entities.Feeling
+import java.io.File
+import java.io.FileOutputStream
+import java.text.SimpleDateFormat
+import java.util.*
+
+
+class SecondActivity: AppCompatActivity(){
+    val calendar = Calendar.getInstance()
+
+    var photo: File? = null
+    var pathPhoto: String? = null
+    var  mood: Feeling?= null
+    val repo = NoteRepository.get()
+
+
+    val TAG = "xyz"
+    private val TAKE_PHOTO_PERMISSION_REQUEST_CODE = 1
+
+
+    fun saveNote(view: View){
+        val gratefulnessEdit = findViewById<TextView>(R.id.gratefulness_edit)
```

Comment:
you don't need to use 'find by view', just write
val gr.. = gratefulness_edit.text
when you will start typing 'grateful....' it will show you an import from the activity layout